enterprise/backend/src/metabase_enterprise/audit_app/pages/common.clj

```diff
@@ -93,7 +93,7 @@
   ;; another
   (let [timezone (memoize/ttl sql-jdbc.sync/db-default-timezone :ttl/threshold (u/hours->ms 1))]
     (fn []
-      (timezone (mdb/db-type) (db/connection)))))
+      (timezone (mdb/db-type) {:datasource mdb.connection/*application-db*}))))
```

Comment:
`(db/connection)` is removed in Toucan 2, but this is what it resolves to anyway.